pkgs/development/python-modules/pyjwt/default.nix

```diff
@@ -29,6 +29,11 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = lib.optionals (lib.versionOlder version "2.0") [
+    # ecdsa changed internal behavior, required if one does 1.7.1 overriding
+    "ec_verify_should_return_false_if_signature_invalid"
```

Comment:
And after merging I noticed the test name is wrong.

```
FAILED tests/contrib/test_algorithms.py::TestEcdsaAlgorithms::test_ec_verify_should_return_false_if_signature_invalid
```

Reply:
And even after patching this home-assistant doesn't build.

```diff
diff --git a/pkgs/development/python-modules/pyjwt/default.nix b/pkgs/development/python-modules/pyjwt/default.nix
index 104dadbd396..b2e642dd5cf 100644
--- a/pkgs/development/python-modules/pyjwt/default.nix
+++ b/pkgs/development/python-modules/pyjwt/default.nix
@@ -31,7 +31,7 @@ buildPythonPackage rec {
 
   disabledTests = lib.optionals (lib.versionOlder version "2.0") [
     # ecdsa changed internal behavior, required if one does 1.7.1 overriding
-    "ec_verify_should_return_false_if_signature_invalid"
+    "test_ec_verify_should_return_false_if_signature_invalid"
   ];
 
   pythonImportsCheck = [ "jwt" ];
```

```
=========================== short test summary info ============================
FAILED tests/contrib/test_algorithms.py::TestEcdsaAlgorithms::test_ec_verify_should_return_false_if_signature_invalid
====== 1 failed, 178 passed, 8 skipped, 1 xfailed, 140 warnings in 2.02s =======
builder for '/nix/store/q8jhyxiksz54n85sbnrbr05n6ggbxqd1-python3.8-pyjwt-1.7.1.drv' failed with exit code 1
```

I'll have to go shopping now and will check back later.